modules/sql-engine/src/integrationTest/sql/types/char/test_char_character_set.test

```diff
@@ -0,0 +1,60 @@
+# name: test/sql/types/char/test_char_character_set.test
+# description: E021-01 Character string types. CHARACTER SET tests
+# group: [char]
+
+# Incorrect name of charset
+statement error
+CREATE TABLE t_incorrect(C1 CHAR CHARACTER SET UTF_8);
+
+statement ok
+CREATE TABLE t_default (c1 CHAR, c2 CHARACTER, c3 CHAR(3), c4 CHARACTER(3));
+
+statement ok
+CREATE TABLE t_utf8 (c1 CHAR CHARACTER SET UTF8, c2 CHARACTER CHARACTER SET UTF8, c3 CHAR(3) CHARACTER SET UTF8, c4 CHARACTER(3) CHARACTER SET UTF8);
```

Comment:
you cover CHARACTER SET but not cover COLLATION should it be done under different activity ?

Reply:
yes, it's separate F690 feature